bayes_opt/bayesian_optimization.py

```
@@ -13,10 +13,14 @@
 from typing import TYPE_CHECKING, Any
 from warnings import warn
 
+import torch
```

Pylint (3.3.7)

[error] Unable to import 'torch'

(E0401)